core/prop/prop_test.go

```diff
@@ -0,0 +1,25 @@
+package prop_test
+
+import (
+	"github.com/otrego/clamshell/core/prop"
+	"testing"
+)
+
+
+
+
+func TestValidate(t *testing.T) {
+
+	var valid prop.Field = "TB"
+	var invalid prop.Field = "TQB"
+
+	test := prop.Validate(valid)
+	if test != true{
+		t.Errorf("Valid SGF field returned %v; want true", test)
+	}
+
+	test = prop.Validate(invalid)
```

Comment:
probably these should be separate tests.

Reply:
I think this is duplicated now? So just remove

Reply:
Ah, forgot to clean up